apps/web/components/eventtype/EventSetupTab.tsx

```diff
@@ -204,6 +206,28 @@ export const EventSetupTab = (
             )}
           </ul>
         )}
+        {validLocations.some((location) => location.type === "integrations:google:meet") && (
+          <div className="flex">
+            <Icon.FiAlertTriangle className="mr-4" />
+            <p className="text-sm">
+              In order to use Google Meet you must set your{" "}
+              <a
+                href="javascript:;"
+                onClick={() => router.push(`/apps/installed/calendar`)}
+                className="underline">
+                default destination calendar{" "}
+              </a>
+              or the{" "}
+              <a
+                href="javascript:;"
+                onClick={() => router.push(`/event-types/${eventType.id}/?tabName=advanced`)}
+                className="underline">
+                event type&#39;s destination calendar
+              </a>{" "}
```

Comment:
We should wrap these in `<Link/>` from `next/router`  to handle the next routing a bit better than having to do the href trickery

Reply:
oh yeah don't do this. good find @sean-brydon

Reply:
Links wrapped in a `<Trans>` component so we should be good.